<launch>
	<arg name="path_save"   default="/home/pi/bags"/>
	<arg name="file_name"  	default="test.bag"/>

	<group ns="record">
		<param name="path_save" type="str" value="$(arg path_save)"	/>
		<param name="file_name" type="str" value="$(arg file_name)"	/>
		<node name="record_ros" pkg="record_ros" type="record_ros" output="screen" respawn="true"/>
	</group>

	<node pkg="xbee_ros_node" type="xbee_mav" name="xbee_mav" args="slave swarm" output="screen" />
	<param name="Xbee_In_From_Buzz" type="str" value="outMavlink" />
	<param name="Xbee_Out_To_Buzz" type="str" value="inMavlink" />
	<param name="Xbee_In_From_Controller" type="str" value="xbee_cmd" />
	<param name="No_of_dev" type="int" value="3" />
	<param name="Xbee_Out_To_Controller" type="str" value="mav_dji_cmd" />

	
	<node name="rosbuzz_node" pkg="rosbuzz" type="rosbuzz_node" respawn="false" output="screen" >	
		<rosparam file="/home/pi/ros_catkin_ws/src/ROSBuzz/launch/launch_config/solo.yaml"/>
		<param name="bzzfile_name" value="/home/pi/ros_catkin_ws/src/ROSBuzz/src/testalone.bzz" />
	    <param name="rcclient" value="true" /> 
		<param name="rcservice_name" value="/buzzcmd" />
		<param name="in_payload" value="/inMavlink"/>
		<param name="out_payload" value="/outMavlink"/>
		<param name="robot_id" value="3"/>
		<param name="No_of_Robots" value="3"/>
		<param name="stand_by" value="$(env ROS_WS)/src/ROSBuzz/src/stand_by.bo"/>
	</node>

</launch>

